app/src/pages/project/AnnotationSummary.tsx

```diff
@@ -189,6 +189,7 @@ export function SummaryValue({
   labelFractions,
   size = "M",
   disableAnimation = false,
+  meanScoreFallback,
```

Comment:
Can you add an example of what this is? I think I vaguely get it but without a docstring I'm not sure when or why I would set it.